### PR TITLE
Fix ESlint issues

### LINKED ISSUE #660: Fix ESLint reported issues
Handle ESLint reported issues and warnings: https://github.com/SimplQ/simplQ-frontend/runs/3791258386

### PR DESCRIPTION
**Pull request by ArturBa**
Closes #660

### CODE REVIEW

**Review by daltonfury42 — APPROVED** [review]
Merging in a few hours, as a client is using dev.simplq instead of main site.

@ArturBa Thanks for the PR, happy hacking!  Don't forget to star our project and suggest it to anyone who wants to manage crowd using a smartphone.

File: simplq/src/components/common/QrScanner/QrScanner.jsx [hunk 1]
```diff
@@ -11,10 +11,10 @@ export default () => {
 
   const handleScan = (data) => {
     if (data != null) {
-      const res = this.getRoute(window.location.origin, data.text);
+      const res = getRoute(window.location.origin, data.text);
```
Comment (maaverik) [inline simplq/src/components/common/QrScanner/QrScanner.jsx:14]: This file is a problem, right? Where are props and getRoute coming from?

File: simplq/src/components/common/QrScanner/QrScanner.jsx [hunk 2]
```diff
@@ -11,10 +11,10 @@ export default () => {
...
 
       if (res.verdict) {
-        this.props.history.push(res.targetRoute);
+        props.history.push(res.targetRoute);
```
Comment (daltonfury42) [inline simplq/src/components/common/QrScanner/QrScanner.jsx:17]: Can you change line 7 to `export default ({history}) => {`?

File: simplq/src/components/common/QrScanner/QrScanner.jsx (hunk 1 above)
Comment (daltonfury42) [reply simplq/src/components/common/QrScanner/QrScanner.jsx:14]: Good callout.

**Review by daltonfury42 — CHANGES_REQUESTED** [review]

File: simplq/src/components/common/QrScanner/QrScanner.jsx (hunk 2 above)
Comment (ArturBa) [reply simplq/src/components/common/QrScanner/QrScanner.jsx:17]: Hi, thanks for this hint. I'm not a React dev :D 
This seems to work :D

File: simplq/src/components/common/QrScanner/QrScanner.jsx (hunk 1 above)
Comment (ArturBa) [reply simplq/src/components/common/QrScanner/QrScanner.jsx:14]: `getRoute` has an issue in jsx, also in the master branch. 
I'm trying to guess what this method should do and where is it defined

Comment (maaverik) [reply simplq/src/components/common/QrScanner/QrScanner.jsx:14]: Yeah, this looks like a miss during a previous refactor. You can skip changing this file for now till we figure out how to handle it better. So can you revert changes to this file and just add changes for the other one?

Comment (ArturBa) [reply simplq/src/components/common/QrScanner/QrScanner.jsx]: Ok, I've revert changes of this file changing only the extension :wink:

**Review by daltonfury42 — APPROVED** [review]
LG

**Review by maaverik — APPROVED** [review]

**Pull request merged**